toptal/picasso
# fix(ButtonAction): loading state for button without icon

No JIRA ticket.

### Description

[🖼️ Abstract design](https://share.goabstract.com/d3a93b64-ab2c-49b0-8a45-0ff20b3cc0f5?mode=design&sha=7507b3ef36c141b7dea49c96ae280417525a1912) | [Design JIRA ticket](https://toptal-core.atlassian.net/browse/FX-1339)

This PR fixes the loading state for action buttons that do not have icons provided.

### How to test

- open the temploy at https://picasso.toptal.net/SPC-NULL-fix-button-action-styles/?path=/story/components-folder--button
- make sure that `Button.Action` has its **loading** state displayed properly

### Screenshots

![actionbuttons3](https://user-images.githubusercontent.com/1390758/96226426-2e053800-0fc5-11eb-84b6-e8f6eb0c1e66.gif)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
